docs/cli/reference.mdx

```diff
@@ -93,7 +45,7 @@ docker run $(wave -i alpine --layer new-layer) sh -c hello.sh
 ```
 </details>
 
-### Build a container from Conda packages
+## Build a container from Conda packages
```

Comment:
We refer to "conda" in other parts of the docs and in the official docs it's referred to as "conda" (https://www.anaconda.com/guides/conda-package-manager-for-data-sciences-ml-and-ai).